velox/expression/CastExpr-inl.h

```diff
@@ -239,21 +239,26 @@ VectorPtr CastExpr::applyDecimalToIntegralCast(
     const TypePtr& toType) {
   using To = typename TypeTraits<ToKind>::NativeType;
 
+  const auto& queryConfig = context.execCtx()->queryCtx()->queryConfig();
```

Comment:
Maybe remove this variable if it is used only once?

Reply:
Update.